check-dependent-jobs/action.yml

```diff
@@ -0,0 +1,28 @@
+name: check-dependent-jobs
+description: |
+  Evaluates the combined the status results of the provided needs context.
+inputs:
+  needs:
+    required: true
+    description: needs context as JSON string
+outputs:
+  is-success:
+    description: The evaluated result of all provided jobs in the needs context.
+    value: ${{ steps.test.outputs.is-success }}
+  status:
+    description: One of success or failure.
+    value: ${{ steps.test.outputs.status }}
+runs:
+  using: composite
+  steps:
+    - id: test
```

Comment:
Do you think we should support this step as OS agnostic?

Reply:
I've just found some issues when the needs contains multiple outputs, and therefore the bash fails

Reply:
Can you please elaborate on this?

Reply:
https://github.com/elastic/oblt-actions/pull/16/commits/0e8037f0f1e1166e82fc37a50c7302dd104fe3ca